### ISSUE TITLE
[ExpandCollapse] Show | Hide all - Collapsible sections

### ISSUE THREAD

**Issue by owestinTelus** [telus/tds-core#452]
## Description

- An accessibility best practice for collapsible sections is to give the user the option to show all items
- Showing all items with a single input (click or key) makes it easier to navigate and read the content 

## Recommendation

- Include ability to expand and collapse all items
- Feature should be available before and after the collapsible sections

## Reference

[Inclusive-Components: Collapsible sections #expand collapse all](https://inclusive-components.design/collapsible-sections/#expandcollapseall)

## Meta

- TDS version: v0.33.1
- Impact: Medium

**Comment by lucylist** [comment]
confirmed with Oskar on this ticket. it's a nice to have and not a must have. Best practice as opposed to fail on a11y
here's an example of it live
http://www.ttc.ca/Customer_Service/Customer_Charter/2018_charter.jsp

I will share it with the design leads as a pattern to see if this is something we want to adopt. if so, it can be an option added to the expand/collapse component

**Comment by theetrain** [comment]
Follow-up: we are accepting contributions for this, but is low on our team's backlog priority.
